cpp/src/aztec/honk/pcs/shplonk/shplonk_multi.hpp

```diff
@@ -136,7 +139,7 @@ template <typename Params> class MultiBatchOpeningScheme {
             for (const auto& [queries_k, openings_k] : multi_claims) {
                 Fr eval{ Fr::one() };
                 for (const Fr& x : queries_k) {
-                    eval *= (r - x);
+                    eval *= (zeta - x);
```

Comment:
It's not a zeta tho.

Reply:
What about `zee` or `z_challenge` or `challenge_z`?

Reply:
Yeah, I guess if we we renamed `u`'s too then every challenge has a Greek name.

Reply:
Yeah I think we have `u, r, z` right now. I like `*_challange` with identical string labels in the manifest.

Reply:
(Challange being the French spelling of course)

Reply:
`zeta` is taken as the point of `pow_zeta`. Going to revert to z_challenge